src/robot2017/src/drive_node.cpp

```diff
@@ -6,35 +6,14 @@
 #include <string>
 #include "robot_msgs/Teleop.h"
 #include "motors.h"
-#include "teleop_exec.h"
+#include "robot_exec.h"
 
 const int refreshRate = 1;
 
 bool onBeagleBone = true;
+bool autState = false;
 ros::Publisher pub;
-TeleopExec teleop;
-
-void teleopReceived(const robot_msgs::Teleop& cmd)
-{
-    std::stringstream message;
-
-    message << "Command recieved: ";
-    message << " " << +cmd.a << " " << +cmd.b << " " << +cmd.x << " " << +cmd.y;
-    message << " " << +cmd.lb << " " << +cmd.rb << " " << +cmd.back;
-    message << " " << +cmd.start << " " << +cmd.l_thumb << " " << +cmd.r_thumb;
-    message << " (" << +cmd.x_l_thumb << " " << +cmd.y_l_thumb << ")";
-    message << " (" << +cmd.x_r_thumb << " " << +cmd.y_r_thumb << ")";
-    message << " (" << +cmd.l_trig << " " << +cmd.r_trig << ")";
-
-    ROS_INFO_STREAM(message.str());
-
-    //function for
-    teleop.teleopExec(cmd);
-
-    // byte range should be [-100, 100]
-    //driveLeft.set(cmd.y_l_thumb * 1.0f / 100.0f);
-    //driveRight.set(cmd.y_r_thumb * 1.0f / 100.0f);
-}
+RobotExec exec = RobotExec();
```

Comment:
fyi you don't need this change - just having RobotExec exec; will call the constructor, whereas this will create two objects and copy one to the other (see http://stackoverflow.com/a/11853771/464886).

Not a big deal, but would like to not rely on copy initialization